src/layout/components/Sidebar/Item.vue

```diff
@@ -22,25 +21,56 @@ export default {
     title: {
       type: String,
       default: ''
+    },
+    isCollapsed: {
+      type: Boolean,
+      default: false
+    },
+    hasChildItems: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    iconMargin() {
+      if (!this.isCollapsed) {
+        return ''
+      }
+      if (this.hasChildItems) {
+        return 'margin-left: -4px;'
+      }
+      return 'margin-left: 18px;'
     }
   }
 }
 </script>
 
 <style scoped>
-.sub-el-icon {
-  width: 1em;
-  height: 1em;
-}
+  .sub-el-icon {
+    width: 1em;
+    height: 1em;
+  }
 
-.item-wrapper {
-  white-space: break-spaces;
-  line-height: 20px;
-  height: auto;
-  margin: 18px 23px 18px 0;
-}
+  .item-wrapper {
+    white-space: break-spaces;
+    line-height: 20px;
+    height: auto;
+    padding: 18px 23px 18px 0;  
```

Comment:
Please delete the spaces at the end of the line as it gives a warning message with the eslint

![compile-with-error](https://user-images.githubusercontent.com/20288327/122397171-b5d18680-cf46-11eb-8144-6e3cd2ed0dd4.png)